apps/staging/mix.exs

```diff
@@ -33,10 +33,10 @@ defmodule Staging.Mixfile do
     [
       {:jsx, "~> 2.8"},
       {:slack, "~> 0.11"},
-      {:ecto, "~> 2.0"},
-      {:postgrex, "~> 0.11"},
+      {:ecto, "~> 3.0"},
+      {:postgrex, "~> 0.15"},
```

Comment:
postrex changelog reports 0.15 requires elixir 1.6 https://github.com/elixir-ecto/postgrex/blob/0f0e70080e55dceca4c0cc6e7a1b81ca835c49b2/CHANGELOG.md?plain=1#L89

we are on 1.4, does this work?
https://github.com/LendingHome/dev_bot/blob/2c87e9e97e19ea65131221220b3b6bdcc3cf9518/elixir_buildpack.config#L3